pallets/pallet-asset-switch/src/xcm/transact.rs

```diff
@@ -87,7 +87,7 @@ where
 			&switch_pair.pool_account,
 			&beneficiary,
 			fungible_amount_as_currency_balance,
-			Preservation::Expendable,
+			Preservation::Preserve,
```

Comment:
Does this now mean that the account must have ED + the amount of remote tokens? Or is that the purpose of `Preservation::Preserve`? What happens if we bring the balance to `ED + 1` and try to remove `2`? Does it go to `ED - 1`? Can it go all the way to `0`? If so, what happens then?

Reply:
This means the account needs at least the ED. If the balance of an account is getting below the ED by the transfer, the `LocalCurrency::transfer` will fail. I think this is exactly what we want, no?

Reply:
Then we need to change the requirement for the pool account to be ED + circulating supply, which is not the case now.

Reply:
So there's no way to not have an ED requirement and still avoid the account from being dusted?

Reply:
https://support.polkadot.network/support/solutions/articles/65000168651-what-is-the-existential-deposit-

Reply:
I think, this is exactly what we need here

Reply:
Yeah but I'm sure the balances pallet has a way to mark accounts as "special", no?